File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        boolean b = false;
        for(int i = 0; i< nums.length; i++){
            
            for(int j = i+1; j<nums.length;j++){
                 if(nums[j] == nums[i]){
                     b = true;
                 }
            }
        }
        return b;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = {}
        for i in nums:
            key = i
            if key not in freq:
                freq[key] = 0
            freq[key] += 1
        top = sorted(freq, key= freq.get, reverse = True)
        n = []
        for i in range(k):
            n.append(top[i])
        return n
            